etc/nix/flake.nix

```diff
@@ -138,6 +138,8 @@
               buildPhase = ''
                 source ${libSh}
                 initPostgres $(pwd)
+                export SECRET_KEY=REALLY_SECRET
```

Comment:
ce64bceda7cde0a9 removed the `DJANGO_DEV=1` setting, I would guess without testing it: Setting neither `DJANGO_DEV=1` nor the `SECRET_KEY` results in an error.

Reply:
Yes, ``DJANGO_DEV`` is not present now. Please use ``make envfile``. A basic make based run is at https://github.com/nexB/vulnerablecode/blob/main/.github/workflows/main.yml

Reply:
@Hritik14 Thanks, if I understand you correctly, you suggest to run ``make dev envfile`` (https://github.com/nexB/vulnerablecode/blob/main/.github/workflows/main.yml#L24). The ``dev`` target makes no sense since the Nix setup  doesn't run ``pip -r requirements`` on purpose. As for the ``envfile`` target, it creates the file but uses the non-deterministic ``urandom`` source, which is not feasible in Nix. I'm not convinced it is better to use make than to simply set the env var as I proposed here.